src/user/dashboard/navigation/navigation.js

```diff
@@ -19,146 +22,145 @@ class Navigation extends Component {
-                fill="black"
-                fill-opacity="0.5"
-              />
-            </svg>
-            <NavLink to="/settings" className="link">
-              <b>Settings</b>
-            </NavLink>
-          </ListGroup.Item>
-          <ListGroup.Item style={divStyle2}
-          className={this.props.logout ? "active" : "inactive"}>
-            <Button variant="link" size="sm" className="log-button" onClick={
-              ()=>this.setState({logout:true})}>
-            <img class="logout" src={logo} alt="L"></img>
-            <b>Logout</b>
-            </Button>
-            <Logout show={this.state.logout}
-              onHide={cancel} />
-          </ListGroup.Item>
-          <ListGroup.Item>
-            <Button variant="outline-secondary" size="sm" className="jitsi" onClick={
-              () => this.setState({open:true})}>
-            <b>Jitsi Meet</b>
-            </Button>
-            {this.state.open ? <Info show={this.state.open}  onHide={close}/> : null}
-          </ListGroup.Item>
-
-        </ListGroup>
+        <List className="list-group">
+          <Link to="/dashboard">
+            <ListItem button>
+              <ListItemIcon className="donut-title">
+                <DonutTitleSmall />
+              </ListItemIcon>
+            </ListItem>
+          </Link>
+          <Link to="/dashboard" className="route">
+            <ListItem button className={this.props.dashboard ? "active" : "inactive"}>
+              <ListItemIcon>
+                <svg
+                  width="18"
+                  height="18"
+                  viewBox="0 0 18 18"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                  className="icon"
+                >
+                  <path
+                    d="M0 10H8V0H0V10ZM0 18H8V12H0V18ZM10 18H18V8H10V18ZM10 0V6H18V0H10Z"
+                    fill="black"
+                    fill-opacity="0.5"
+                    className="path-name"
+                  />
+                </svg>
+              </ListItemIcon>
+              <span className="link-text">Dashboard</span>
```

Comment:
add the property text-decoration to null, as on hover giving the underline and also use the same font-style for all the options on the sidebar.
make settings and logout option sticky to that point.